src/main/java/org/opentripplanner/netex/mapping/StopTimesMapper.java

```diff
@@ -168,17 +168,23 @@ private StopTime mapToStopTime(
         stopTime.setStop(stop);
         if (passingTime.getArrivalTime() != null || passingTime.getDepartureTime() != null) {
             stopTime.setArrivalTime(calculateOtpTime(
-                passingTime.getArrivalTime(),                 
-                passingTime.getArrivalDayOffset(),
-                passingTime.getDepartureTime(),
-                passingTime.getDepartureDayOffset()
+                    passingTime.getArrivalTime(),
+                    passingTime.getArrivalDayOffset(),
+                    passingTime.getDepartureTime(),
+                    passingTime.getDepartureDayOffset()
             ));
             stopTime.setDepartureTime(calculateOtpTime(
-                passingTime.getDepartureTime(),
-                passingTime.getDepartureDayOffset(),
-                passingTime.getArrivalTime(),
-                passingTime.getArrivalDayOffset()
+                    passingTime.getDepartureTime(),
+                    passingTime.getDepartureDayOffset(),
+                    passingTime.getArrivalTime(),
+                    passingTime.getArrivalDayOffset()
             ));
+
+            // From NeTEx we define timepoint as a waitpoint with waiting time defined (also 0)
+            if (Boolean.TRUE.equals(stopPoint.isIsWaitPoint())
+                    && passingTime.getWaitingTime() != null) {
+                stopTime.setTimepoint(1);
```

Comment:
I think it would be an immprovment if the OTP internal model for time point would be an enum.  The value 1 here is not telling me what this is. Then there is also a place to document what this mean in NeTEx as well. So for the future don´t hasitate to make this kind of improvements. For example: `enum TimePoint {ESTIMATE(0), TIMEPOINT(1); int gtfsCode;}`.